### PR TITLE
BART: Categorical example

### PR DESCRIPTION
**Pull request by PabloGGaray**
Closes https://github.com/pymc-devs/pymc-bart/issues/100

----
📚 Documentation preview 📚: https://pymc-examples--663.org.readthedocs.build/en/663/

### CODE REVIEW

**Comment by PabloGGaray** [comment]
<p>It's ok the "3" in "but fitting 3 independent trees."?</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/663/discussion/#comment-2127184143'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by aloctavodia** [comment]
<p>Well, it is 3 independent "sum of trees". Better to remove the "3"</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/663/discussion/#comment-2127184143'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by PabloGGaray** [comment]
<p>Yes, it's a copy/paste error, thanks.</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/663/discussion/#comment-2127539066'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by aloctavodia — APPROVED** [review]
LGTM. What do you think @fonnesbeck?

File: examples/bart/bart_categorical_hawks.myst.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+import pymc_bart as pmb
+import seaborn as sns
+
+warnings.simplefilter(action="ignore", category=FutureWarning)
+```
+
+```{code-cell} ipython3
+# set formats
+RANDOM_SEED = 8457
+rng = np.random.RandomState(RANDOM_SEED)
```
[the remark this answers is not in the document]
Comment (OriolAbril) [reply examples/bart/bart_categorical_hawks.myst.md:44]: I am not sure the rng variable is used though

[the remark this answers is not in the document]
Comment (PabloGGaray) [reply examples/bart/bart_categorical_hawks.myst.md]: Yes, you are right, `rng` is not used.

**Pull request merged**